### DISCUSSION TITLE
Why do we need sar files generated by sa2 when we have the sa files generated by sa1?

### DISCUSSION THREAD

**Discussion by peregineme** [sysstat/sysstat#380] [Q&A]
We can always open up the sa files using sar or sadf?
Aren't the sar files a waste of space?
The only issue I see is if we cannot open the sa files later because we don't have a suitable machine.

**Comment by sysstat** [comment]
`sarDD` files are plain text files whose contents is the same as what you get on your screen when you use `sar`. You can read them without any special command, edit them, save them for future inspection and they are totally independent from sar's version, which means that you are sure you will be able to read them in a bunch of years.
If you consider them a waste of space, just don't create them, that is, don't run `sa2` script periodically by cron or systemd.

**Reply by peregineme** [reply]
There seems to be a caveat - the `sa2` script does the compression and histroy management of the `saDD` files, and therefore it is needed.
In order not to generate large text files, I run `saDD` at the beginning of the day.

**Reply by sysstat** [reply]
@peregineme
It should just be a matter of setting the `REPORTS` variable to false in the sysstat configuration file (default location is `/etc/sysconfig/sysstat`).
In this way you won't get the reports (`sarDD` files).
